Dockerfile

```diff
@@ -55,14 +55,20 @@ RUN ["dash", "-c", "\
      bash-completion \
      ca-certificates \
      curl \
+     g++ \
```

Comment:
These new packages were brought over from the testing Dockerfile that got merged into one with the examples Dockerfile.